addons/mail/static/tests/helpers/webclient_setup.js

```diff
@@ -172,6 +174,17 @@ async function getWebClientReady(param0) {
             webClientParameters[parameterName] = value;
         }
     }
+
+    const pyEnv = param0.pyEnv;
+    patchWithCleanup(sessionInfo, {
+        user_context: {
+            lang: "en",
+            uid: pyEnv.currentUserId,
+            tz: "taht",
```

Comment:
I think it would be more robust to only change what is needed and to keep the rest intact.
```suggestion
            ...sessionInfo.user_context,
            uid: pyEnv.currentUserId,
```

Reply:
Change done